packages/examples/src/geometry-domain/textbook_problems/ex.sub

```diff
@@ -21,22 +18,19 @@ Angle aXYZ := InteriorAngle(X, Y, Z)
 Angle aYZX := InteriorAngle(Y, Z, X)
 Angle aZXY := InteriorAngle(Z, X, Y)
 
-EqualLengthMarker1(ST, XY)
+EqualLengthMarker(ST, XY)
 EqualLength(ST, XY)
 
-EqualLengthMarker2(XZ, US)
-EqualLengthMarker2(XZ, UV)
+EqualLengthMarker(XZ, US)
 EqualLength(XZ, US)
-EqualLength(XZ, UV)
 
-EqualLengthMarker3(UT, VT)
-EqualLengthMarker3(ZY, UT)
-EqualLength(UT, VT)
+EqualLengthMarker(ZY, UT)
 EqualLength(ZY, UT)
 
-EqualAngleMarker1(aUTS, aZXY)
+EqualAngleMarker(aUTS, aZXY)
 EqualAngle(aUTS, aZXY)
-EqualAngleMarker2(aUTS, aXYZ)
+EqualAngleMarker(aUTS, aXYZ)
 EqualAngle(aUTS, aXYZ)
-EqualAngleMarker3(aTUS, aYZX)
-AutoLabel All
\ No newline at end of file
+EqualAngleMarker(aTUS, aYZX)
```

Comment:
I have a question about this diagram: is the Substance program wrong, or does the Style program just fail to display it properly? Currently the diagram shows that `aUTS` and `aXYZ` are equal, and that `aTUS` and `aYZX` are equal, but it doesn't show that `aUTS` and `aZXY` (and thus also `aXYZ`) are equal, because the one-arc equality marker on `aUTS` gets covered by the two-arc equality marker on the same angle, leaving a lone one-arc equality marker on `aZXY`. Is this simply due to current limitations of the Style language, which we are working on resolving via support for transitivity?

Reply:
Kind of both. A typo in the Substance program first, which accidentally made 3 angles equal to each other. The Style program doesn't handle `EqualAngleMarker` the same way as `EqualLengthMarker` (i.e. setting the mark paths s.t. that look the same). 

I fixed both of the above, but that approach is still flawed when there are `A=B=C` and `D=E` in Substance, which I don't think I can fix due to Style lang limitations.